mappings/WebVR/Oculus Go Controller/mapping.json

```diff
@@ -0,0 +1,99 @@
+{
+    "version" : "0.1.0",
```

Comment:
I assume you want these to be just "0.1" to match with #35?

Reply:
Yeah, I wasn't sure which would get approved first ;)